### PR TITLE
Style changes

### LINKED ISSUE #101: comments and code and new lines
_Remember: the better you describe your issue, the easier it is for us to help you._

This is more a place holder for us to dicuss. not really a wrong or right. 
there are currently 2 conventions happening in the java code.

    /** comment*/
    code
    /** comment*/
    code

and

    /** comment*/
    code

    /** comment*/
    code

(note the new line between new blocks). Persoanlly i find the second more readable than the first, and most of the java ive seen to date does the second, but the comms seems to follow the first. 

discuss if we want a unified convention here, or meh to either. Im not sure, so leaving it as issue for opening debate

### PR DESCRIPTION
**Pull request by dkfellows**
fixes #101 

Style changes are _large_.

### CODE REVIEW

**Comment by alan-stokes** [comment]
failed checks. will review when ya ready (aka when ticks are green)

File: SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/eieio/EIEIODataMessage.java [hunk 1]
```diff
@@ -380,11 +388,17 @@ public int getSize() {
 		}
 
 		private static final int PREFIX_BIT = 7;
```
Comment (alan-stokes) [inline SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/eieio/EIEIODataMessage.java]: These should be at top of file.

File: SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/RouterDiagnostics.java [hunk 2]
```diff
@@ -41,23 +41,32 @@
 public class RouterDiagnostics {
```
Comment (alan-stokes) [inline SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/RouterDiagnostics.java]: i think (but am really only slightly off the fence on this one, that the enum should be at top of file.

File: SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/Signal.java [hunk 3]
```diff
@@ -54,8 +54,10 @@
```
Comment (alan-stokes) [inline SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/Signal.java]: for the ones above this. should there not be new lines between the varable and next comment???

File: SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/SystemVariableDefinition.java [hunk 4]
```diff
@@ -209,11 +209,13 @@
```
Comment (alan-stokes) [inline SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/SystemVariableDefinition.java]: same for the ones above here

File: SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/scp/IPTagGet.java [hunk 5]
```diff
@@ -104,11 +112,17 @@ private Response(ByteBuffer buffer)
 		}
 
 		private static final int IPV4_BYTES = 4;
```
Comment (alan-stokes) [inline SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/scp/IPTagGet.java]: mov above the private response?

File: SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/transceiver/UDPTransceiver.java [hunk 6]
```diff
@@ -152,6 +155,7 @@ public void close() throws Exception {
 	}
 
 	private static final InetAddress WILDCARD_ADDRESS;
```
Comment (alan-stokes) [inline SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/transceiver/UDPTransceiver.java]: this still has the issue of multiple methods between the 2 uses, so psuh to top of class

**Comment by Christian-B** [comment]
The standard which I agree with is that PRIVATE variables even static ones do not need javadocs as long as by their name it is relatively clear what they represent.

File: SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/RouterDiagnostics.java (hunk 2 above)
Comment (dkfellows) [reply SpiNNaker-comms/src/main/java/uk/ac/manchester/spinnaker/messages/model/RouterDiagnostics.java]: Java _mostly_ doesn't care, and reordering class contents is off-topic for this PR.

**Comment by dkfellows** [comment]
This will need to be redone if we want it; `master` has moved too far.

**Pull request closed without merging**